### ISSUE TITLE
Does this coding mean we never override sequence length? (in FileSequence.java)

### ISSUE THREAD

**Issue by jimk-bdrc** [buda-base/asset-manager#122]
```java
 private int getSequenceSubstringLength() {
        if (_sequenceLength == 0) {
            _sequenceLength = PropertyManager.PropertyManagerBuilder().MergeClassResource("/auditTool.properties", getClass()).getPropertyInt(this.getClass().getCanonicalName() + ".SequenceLength");
        }
        return _sequenceLength;
    }
```

**Comment by jimk-bdrc** [comment]
Yes it does. Apparently, only FileSequenceTest is the only test which uses properties. Means that a base class (and the basest of test classes) should get and save it. Or FileSequenceTest gets the value of the property (sequence length) it wants as a parameter.

**Comment by jimk-bdrc** [comment]
Closed in PR #159
